### PR TITLE
Addressing various Ember depreciations required for 4.0

### PR DESCRIPTION
**Pull request by Monkeychip**
- Removed [Ember.logger.](https://deprecations.emberjs.com/v3.x/#toc_ember-console-deprecate-logger) This was used in only one area, the not-found test and it honestly didn't seem to be doing anything. I removed it and the tests continued to pass. I also paused each test to make sure it was functioning as expected. 
- We have `jquery-intergration` set to false in our config, yet it still exists in our package json. Removed it.
- [Prevent](https://deprecations.emberjs.com/v3.x/#toc_ember-string-prototype_extensions) user from using Ember string methods on strings. 
- [Remove the reopen](https://deprecations.emberjs.com/v3.x/#toc_ember-built-in-components-reopen) on linkTo. This involved some scss changes and a restructure on the replication tabs page. Double checked all areas this effected look good.

### CODE REVIEW

File: ui/app/styles/components/tabs.scss [hunk 1]
```diff
@@ -13,20 +13,26 @@
       &:focus {
         box-shadow: none;
       }
-      &.is-active,
-      &.is-active .tab {
+      &.active,
```
Comment (zofskeez) [inline ui/app/styles/components/tabs.scss:16]: This looks good. I wonder why the `activeClass` override was there in the first place? Was there an alternate styling for `.active` that was being applied maybe?

**Review by zofskeez — APPROVED** [review]
This all looks good to me nice work!

File: ui/app/styles/components/tabs.scss [hunk 2]
```diff
@@ -13,20 +13,26 @@
...
+      .tab {
```
Comment (hashishaw) [inline ui/app/styles/components/tabs.scss:17]: I think we still want to only target active tags here, right? 
```suggestion
      &.active .tab {
```

File: ui/app/styles/components/tabs.scss (hunk 1 above)
Comment (Monkeychip) [reply ui/app/styles/components/tabs.scss:16]: Honestly, no idea. But good idea. Let me do some more checking on .active anywhere else.

File: ui/app/styles/components/tabs.scss (hunk 2 above)
Comment (Monkeychip) [reply ui/app/styles/components/tabs.scss:17]: Good catch. I'll amend.

**Pull request merged**